assets/js/map.js

```diff
@@ -108,7 +99,7 @@
   function renderMapLocations(locations) {
     // size accessor (keeping things DRY)
     var size = function getSize(d) {
```

Comment:
Hey @shawnbot, that might be something worth mentioning in the [Testing Cookbook](http://18f.github.io/testing-cookbook) somewhere.